### ISSUE TITLE
The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested

### ISSUE THREAD

**Issue by fugohan** [automatisch/automatisch#800]
I was trying to run docker on my Raspberry Pi 4 and ran into this warning/error. I thought docker-compose is fetching the correct platform images. 

``` bash
~/docker-compose/automatisch$ sudo docker-compose up -d 
[sudo] password for thisuser: 
Building main
Sending build context to Docker daemon  5.632kB
Step 1/7 : FROM automatischio/automatisch:0.3.0
0.3.0: Pulling from automatischio/automatisch
ca7dd9ec2225: Already exists 
6fb1b25da510: Already exists 
7604515c95e1: Already exists 
e6bf7dff8d85: Already exists 
f1886b033cb5: Already exists 
5227223a2233: Already exists 
5c970ed2a370: Pull complete 
Digest: sha256:c4ca4b6d1117fb271050904125f95097416cdf447097d447698ccc47a533ca55
Status: Downloaded newer image for automatischio/automatisch:0.3.0
 ---> 2bad11811fff
Step 2/7 : WORKDIR /automatisch
 ---> [Warning] The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
 ---> Running in 9643b152302c
Removing intermediate container 9643b152302c
 ---> d574e526fb5d
Step 3/7 : RUN apk add --no-cache openssl dos2unix
 ---> [Warning] The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
 ---> Running in f8b1f4b7d3bb
exec /bin/sh: exec format error
The command '/bin/sh -c apk add --no-cache openssl dos2unix' returned a non-zero code: 1
ERROR: Service 'main' failed to build : Build failed

```

**Comment by barinali** [comment]
Hey @fugohan, thanks for creating the issue. We do not currently support linux/arm64. However, it's an item on our todo list. I believe we'll soon tackle it.

**Comment by barinali** [comment]
@fugohan, we have released version 0.4.0 on Docker Hub with different CPU architectures. Could you please test it out on your raspberry pi 4 and let us know if it works?

**Comment by fugohan** [comment]
It works! Thank you
